lambdas/user_access_patterns/lambda_for_createUser/index.js

```diff
@@ -1,11 +1,6 @@
 const AWS = require("aws-sdk");
 
 //AWS.config.loadFromPath("../../../keys.json");
```

Comment:
I think its better to comment out the local setup code

Reply:
no need to remove comments, we are suppose to push production code not the testing code.

Reply:
> no need to remove comments, we are suppose to push production code not the testing code.

Agreed